### DISCUSSION TITLE
Livewire@2 and AlpineJS@3.9 after livewire update the template alpine data dose not change

### DISCUSSION THREAD

**Discussion by sahakyanArt** [alpinejs/alpine#2719] [7. Bugs (archived)]
After livewire returning different templates with different alpine data on it, the alpine init function works but data properties dose not updated.
Look here: https://laravelplayground.com/#/snippets/fd7f4773-5312-466f-a217-9d842f617e99

**Comment by jskorlol** [comment]
So am I.

As of 3.9, @event cannot be detected.
